src/tools/replay/nodes/RecordEventNodes.java

```diff
@@ -1,13 +1,12 @@
 package tools.replay.nodes;
 
 import tools.replay.TraceRecord;
-import tools.replay.actors.ActorExecutionTrace.ActorTraceBuffer;
+import tools.replay.actors.UniformExecutionTrace.UniformTraceBuffer;
 
 
 public final class RecordEventNodes {
   public static final int ONE_EVENT_SIZE   = 1 + Long.BYTES;
-  public static final int TWO_EVENT_SIZE   = 1 + (2 * Long.BYTES);
-  public static final int THREE_EVENT_SIZE = 1 + (3 * Long.BYTES);
+  public static final int THREE_EVENT_SIZE = 1 + Long.BYTES + Long.BYTES + Long.BYTES;
```

Comment:
This is used in only one place in the `TraceParser`. Looks like something that is not used anymore?